cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/extensions/CgmesControlAreaAdderImpl.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.extensions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.Network;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public class CgmesControlAreaAdderImpl extends AbstractExtensionAdder<Network, CgmesControlAreaMapping> implements CgmesControlAreaAdder {
+
+    private Map<String, CgmesControlArea> cgmesControlAreas = new HashMap<>();
+
+    public CgmesControlAreaAdderImpl(Network extendable) {
+        super(extendable);
+    }
+
+    @Override
+    public CgmesControlArea newCgmesControlArea(String controlAreaId, String controlAreaName, String energyIdentCodeEic, double netInterchange) {
```

Comment:
rather than `newCgmesControlArea`, maybe call it `addCgmesControlArea`, the `new` prefix in the API generally means it returns an adder. The other solution would be to create a full adder but I think it is too much...